script/Arcueid/Day 5/05_00_ARC05_3_2B.txt

```diff
@@ -6,7 +6,7 @@
 [sha:b9ffb48a4d1a0dcdceeaf6c4a2b6a2d99cfdb51b]{
 -- Page 10007, Offset 9202.
 -- お腹減ってるでしょ、志貴？　さっきのおかわり、いる？　新しいの作ってあげよっか？」
-You must be hungry, right Shiki?
+You must be hungry, right, Shiki?
```

Comment:
Not sure about this one, I don't really feel like I pause here when I say this line